welcome/run-d-program-locally.md

```diff
@@ -0,0 +1,40 @@
+# Eseguire programmi D sul proprio computer
+
+Il linguaggio D è accompagnato non solo dal compilatore `dmd`, ma anche dallo pseudo-interprete `rdmd` e dal gestore di pacchetti `dub`.
+
+### Il compilatore DMD
+
+Il compilatore *DMD* compila file D per produrre un file binario eseguibile.
+Sulla riga di comando *DMD* dev'essere invocato seguito dal/dai file da compilare:
+
+    dmd esempio.d
+
+Il comportamento del compilatore può essere modificato tramite le molte opzioni specificabili.
+L'elenco delle opzioni disponibili si può trovare nella [documentazione online](https://dlang.org/dmd.html#switches) o eseguendo `dmd --help`.
+
+### Compilazione al volo con `rdmd`
+
+Il programma `rdmd`, distribuito congiuntamente al compilatore DMD,
+garantisce che il file e tutte le sue dipendenze siano compilate ed esegue automaticamente l'applicazione prodotta:
+
+    rdmd esempio.d
+
+Su sistemi UNIX la riga shebang `#!/usr/bin/env rdmd`, posta come prima riga di un file D, ne permette l'esecuzione diretta come se si trattasse di uno script.
+
+Browse the [online documentation](https://dlang.org/rdmd.html) or run `rdmd --help` for for an overview of available flags.
+
+### Il gestore di pacchetti `dub`
+
+D's standard package manager is [`dub`](http://code.dlang.org). When `dub` is
+installed locally, a new project `hello` can be created using
+the command line:
+
+    dub init hello
+
+Running `dub` inside this folder will fetch all dependencies, compile the
+application and run it.
+`dub build` will compile the project.
+
+Browse the [online documentation](https://code.dlang.org/docs/commandline) or run `dub help` for an overview of available commands and features.
+
+All available dub packages can be browsed through dub's [web interface](https://code.dlang.org).
```

Comment:
Well, I didn't really want to push this file, but another one, namely #3.
But given that I made this error, I decided to create the WIP label and put it on this, so I remember it's not finished. Sorry for the noise.